environments/base/manifests/thanos-receive-default-service.yaml

```diff
@@ -15,13 +15,13 @@ spec:
   ports:
   - name: grpc
     port: 10901
-    targetPort: 10901
+    targetPort: grpc
```

Comment:
Let's be consistent with target ports and use the actual port number not the name or vice versa.

Reply:
That's actually a kube-thanos change and I made this after looking at the Kubernetes docs on Services.
I guess we want the port _number_ for both port and targetPort then?

Reply:
Yes, I believe it's simpler and straightforward. However, if it's the _recommended_ way of doing it then let's use it everywhere.

Reply:
It's not very clear and I feel like there's no definite answer. However, now I see that the docs say:

> Note: A Service can map any incoming port to a targetPort. By default and for convenience, the targetPort is set to the same value as the port field.

I'll move to `port == targetPort` then :man_shrugging: :smile: